aiida/cmdline/commands/work.py

```diff
@@ -8,12 +8,19 @@
 # For further information please visit http://www.aiida.net               #
 ###########################################################################
 import click
+import logging
 from tabulate import tabulate
 from aiida.cmdline.commands import work, verdi
 from aiida.cmdline.baseclass import VerdiCommandWithSubcommands
 
 CONTEXT_SETTINGS = dict(help_option_names=['-h', '--help'])
 
+LOG_LEVELS = [
+    levelname for levelname in [
+        logging.getLevelName(i) for i in range(51)
```

Comment:
I'll make it such that it will at least report user-defined higher levels, but they cannot be selected as inputs.